mockserver/src/main/groovy/pl/touk/mockserver/server/Mock.groovy

```diff
@@ -148,6 +151,17 @@ class Mock implements Comparable<Mock> {
         }
     }
 
+    void setMaxUses(Integer maxUses) {
+        if (maxUses > 0) {
+            this.maxUses = maxUses
+            this.usesLeft = maxUses
+        }
+    }
+
+    void setCyclic(Boolean cyclic) {
+        this.cyclic = cyclic
```

Comment:
don't keep null - use elvis to have always true or false

Reply:
Done.